src/regression_tracker.py

```diff
@@ -38,19 +35,25 @@ def _create_regression(self, failed_node, last_successful_node):
         regression = {}
         for field in self._regression_fields:
             regression[field] = failed_node[field]
-        regression['parent'] = failed_node['id']
-        regression['regression_data'] = [last_successful_node, failed_node]
-        self._api_helper.submit_regression(regression)
+
+        regression['kind'] = 'regression'
```

Comment:
This line can be eliminated as the default value for `Regression.kind` is already `regression`.